src/util/__tests__/createServerRootMixin.test.js

```diff
@@ -214,7 +231,7 @@ describe('createServerRootMixin', () => {
           ])
         ),
         serverPrefetch() {
-          return this.instantsearch.findResultsState(this);
+          return this.instantsearch.findResultsState(this, renderToString);
```

Comment:
is this the best signature, should it maybe be `{ component: this, renderToString }`?

Reply:
yeah it looks better!